src/core.js

```diff
@@ -56,14 +59,14 @@ define([
     datasources,
     util,
     jrUtil,
+    codemirror,
     debug
 ) {
     
     // Load these modules in the background after all runtime dependencies have
     // been resolved, since they're not needed initially.
     setTimeout(function () {
         require([
-            'codemirror',
```

Comment:
Yep. I was able to once I figured out the other issues with the build https://github.com/dimagi/Vellum/pull/772/commits/b3bca8a4936b7404da12579f4e26cc38de18423e